lfs/gitscanner.go

```diff
@@ -134,6 +174,17 @@ func (s *GitScanner) opts(mode ScanningMode) *ScanRefsOptions {
 	return opts
 }
 
+func firstGitScannerCallback(callbacks ...GitScannerCallback) (GitScannerCallback, error) {
+	for _, cb := range callbacks {
+		if cb == nil {
+			continue
+		}
+		return cb, nil
+	}
+
+	return nil, errors.New("No callback given")
```

Comment:
We talked about this in 🍖 -space, but this would be a good candidate to extract to a sentinel value if you want to keep the `firstGitScannerCallback` business 👍